packages/console/src/components/CreateTenantModal/index.tsx

```diff
@@ -122,18 +126,21 @@ function CreateTenantModal({ isOpen, onClose }: Props) {
               render={({ field: { onChange, value, name } }) => (
                 <RadioGroup type="small" name={name} value={value} onChange={onChange}>
                   {/* Manually maintaining the list of regions to avoid unexpected changes. We may consider using an API in the future. */}
-                  {[RegionName.EU, RegionName.US].map((region) => (
-                    <Radio
-                      key={region}
-                      title={
-                        <DangerousRaw>
-                          <Region regionName={region} />
-                        </DangerousRaw>
-                      }
-                      value={region}
-                      isDisabled={isSubmitting}
-                    />
-                  ))}
+                  {[RegionName.EU, RegionName.US, RegionName.AU].map(
```

Comment:
nit: can use `Object.values(RegionName)`? If we have a lot of regions, we do not need to enumerate all of them in this case.

Reply:
can you check the comment above?